### PR TITLE
pep8 fixes

### PR DESCRIPTION
**Pull request by pazz**
This mostly shortens lines down to <=79 chars and fixes some other small
things I found using the pep8 tool.

### CODE REVIEW

File: alot/widgets/search.py [hunk 1]
```diff
@@ -115,7 +115,8 @@ def pad(string, shorten=None):
             if self.thread:
                 fallback_normal = struct[name]['normal']
                 fallback_focus = struct[name]['focus']
-                tag_widgets = sorted(TagWidget(t, fallback_normal, fallback_focus)
+                tag_widgets = sorted(TagWidget(t, fallback_normal,
+                                               fallback_focus)
```
Comment (dcbaker) [inline alot/widgets/search.py]: This is really hard to read. What about
```python
tag_widgets = sorted(
    TagWidget(t, fallback_normal, fallback_focus)
    for t in self.thread.get_tags())
```

File: alot/commands/globals.py [hunk 2]
```diff
@@ -78,7 +78,9 @@ def apply(self, ui):
 
         if ui.db_was_locked:
             msg = 'Database locked. Exit without saving?'
-            if (yield ui.choice(msg, msg_position='left', cancel='no')) == 'no':
+            if (yield ui.choice(msg,
+                                msg_position='left',
+                                cancel='no')) == 'no':
```
Comment (dcbaker) [inline alot/commands/globals.py]: I've been thinking about these, they're hard to read, but what if we took it out of the if statement?
```python
response = yield ui.choice(msg, msg_position='left', cancel='no')
if responce == 'no':
    ...
```

File: alot/widgets/search.py (hunk 1 above)
Comment (pazz) [reply alot/widgets/search.py]: ok

File: alot/commands/globals.py (hunk 2 above)
Comment (pazz) [reply alot/commands/globals.py]: works for me

**Comment by dcbaker** [comment]
This looks good to me at this point. I don't know what you want to do about the codeclimate warnings, I looked at them and they all look pretty silly to me.

**Comment by pazz** [comment]
I'll fix the 'newline before bin op' issues and ignore the rest.

**Pull request merged**